github3/issues/issue.py

```diff
@@ -240,6 +242,13 @@ def is_closed(self):
             return True
         return False
 
+    def is_locked(self):
+        """Checks if the issue is locked.
+
+        :returns: bool
+        """
+        return self.locked
```

Comment:
> Will this eventually have an API call?

Not as far as I know. 
I've made another PR, with a clean history, based on your feedback: https://github.com/sigmavirus24/github3.py/pull/574
Is that the right way to approach this?